tests/virt_autotest/login_console.pm

```diff
@@ -263,6 +263,16 @@ sub login_to_console {
 sub run {
     my $self = shift;
     $self->login_to_console;
+    #julie
+    script_run("date");
+    script_run("systemctl status sshd");
+    save_screenshot;
+    script_run("systemctl restart sshd");
+    save_screenshot;
+    sleep 5;
+    script_run("systemctl status sshd");
+    save_screenshot;
+    use_ssh_serial_console;
```

Comment:
Suggest adding such new functionality involving multiple lines with existing or new subroutine.

Reply:
Sorry, this PR was incorrect. Closing....  The root cause is https://github.com/SUSE/qa-automation/pull/822